src/components/Layout/ScrollableParent.tsx

```diff
@@ -16,9 +16,11 @@ const ScrollableParentContext = createContext<ScrollableParentContextProps>({
 // Allow any css to be applied to the ScrollableParent container.
 interface ScrollableParentContextProviderProps {
   xss?: Properties;
+  tagName?: keyof JSX.IntrinsicElements;
```

Comment:
Just curious, but maybe throw in a `// I.e. for blueprint we use tagName of x for reason y` for future maintainers.